desktop_version/src/editor.cpp

```diff
@@ -355,6 +355,8 @@ void editorclass::reset()
     hookmenu=0;
     numhooks=0;
     script.customscript.clear();
+
+    returneditoralpha = 0;
```

Comment:
I get that you're trying to make this initialization consistent with the rest of the initializations, and want to reset it properly, too, but why touch this in this PR? (Also touched in `editor.h`.)

Reply:
I am using an old compiler and could not build with it initialized in the header.

Reply:
As @flibitijibibo said on #47:

> Try to aim for the absolute minimum C++ standard possible (if you can get away with it, write it as basically C). We have to support some very old compilers, so there’s no room to be overly fancy. [link](https://github.com/TerryCavanagh/VVVVVV/pull/47#issuecomment-573376232)

Reply:
Alright, well, put it in another PR, then.

Reply:
I can do it as a separate PR if you want. Another problem: #47 introduced the use of `char32_t` which only works on compilers that support C++11. I could roll fixes for both into one "compatibility" PR.

Reply:
Done. Undid that change

Reply:
Yeah, fix the `char32_t` stuff and make it a general "fix compatibility" PR.

Reply:
Sounds good